### ISSUE TITLE
Can we achieve Zero down time when there is setup:upgrade is happening 

### ISSUE THREAD

**Issue by ghost** [stu-tu/aks-magento#1]
Can we achieve Zero down time when there is setup:upgrade is happening

**Comment by fztu** [comment]
No. In this setup, the build phase is independent. However, during the setup:upgrade, the site will still have downtime.
